.github/CONTRIBUTING.md

```diff
@@ -1,338 +1,22 @@
-<div align="center">
-    <img src="https://github.com/PreMiD.png?size=2048" width="128px" style="max-width:100%;">
-    <h3 style="font-size: 2rem; margin-bottom: 0">Presence Guidelines</h3>
-    <h4 style="margin-top: 0">Revision 3</h4>
-    <br />
-</div>
+# Contrubutions
 
-# Guidelines
+As an open-sourced project, we welcome contributions with open arms. 
 
-When publishing Presences to this GitHub, we require you to follow a set of guidelines. To some, these strict rules may seem harsh. However, the implementation of these rulesets will keep us and the users from running into any issues.
+> You should ensure you read the entirety of this document before contributing to the repository. We require the following rules for good practice. 
 
-# Creation
+## Issues 
+Issues are designed for tracking bugs, feature requests and service requets by using their associated templates. The content of each field should be detiailed thoroughly before being submitted. 
```

Comment:
an issue is not a contribution, this file should be about making presences, testing them and opening PRs

Reply:
Yes it is

Reply:
No it isn’t.

Reply:
Github class it as a contribution
https://github.com/actions/runner/blob/main/docs/contribute.md